packages/pwa/app/commerce-api/auth.js

```diff
@@ -278,10 +278,16 @@ class Auth {
         }
 
         const response = await this._api.shopperLogin.authorizeCustomer(options, true)
-
         if (response.status >= 400) {
-            const json = await response.json()
-            throw new HTTPError(response.status, json.message)
+            let text = await response.text()
+            let errorMessage = text
+            try {
+                const data = JSON.parse(text)
+                if (data.message) {
+                    errorMessage = data.message
+                }
+            } catch {} // eslint-disable-line no-empty
+            throw new HTTPError(response.status, errorMessage)
```

Comment:
The previous error handling assumed the error response was JSON with an attribute `message`.

That might be the case for some errors, but certainly isn't the case for ALL errors.

This change better guards against a wider set of issues.